sql/core/src/test/scala/org/apache/spark/sql/util/DataFrameCallbackSuite.scala

```diff
@@ -227,9 +227,7 @@ class DataFrameCallbackSuite extends QueryTest
         spark.range(10).select($"id", $"id").write.insertInto("tab")
       }
       sparkContext.listenerBus.waitUntilEmpty()
-      assert(exceptions.length == 1)
-      assert(exceptions.head._1 == "insertInto")
-      assert(exceptions.head._2 == e)
+      assert(exceptions.length == 0)
```

Comment:
This is a breaking change? We can't see errors now?

Reply:
It's no chance to create `SparkListenerSQLExecutionEnd` and post to `ExecutionListenerBus`.
We can see errors at exception stack now.